polkadot/xcm/src/lib.rs

```diff
@@ -634,6 +634,9 @@ pub mod prelude {
 		VersionedAssetId, VersionedAssets, VersionedInteriorLocation, VersionedLocation,
 		VersionedResponse, VersionedXcm, WrapVersion,
 	};
+
+	/// The minimal supported XCM version
+	pub const MIN_XCM_VERSION: XcmVersion = 3;
```

Comment:
@raymondkfcheung @franciscoaguirre please check this comment: https://github.com/paritytech/polkadot-sdk/pull/7438/files#r1981291318

Reply:
the discussion above is relating to changes in master, don't think it's relevant for backports (where the versions never change anymore)